tests/UKHO.ERPFacade.API.FunctionalTests/FunctionalTests/LicenceUpdatedWebhookScenario.cs

```diff
@@ -0,0 +1,90 @@
+﻿using FluentAssertions;
+using NUnit.Framework;
+using UKHO.ERPFacade.API.FunctionalTests.Configuration;
+using UKHO.ERPFacade.API.FunctionalTests.Helpers;
+using UKHO.ERPFacade.API.FunctionalTests.Service;
+
+namespace UKHO.ERPFacade.API.FunctionalTests.FunctionalTests
+{
+    [TestFixture]
+    public class LicenceUpdatedWebhookScenario
+    {
+        private LicenceUpdatedEndpoint _LUpdatedWebhookEndpoint { get; set; }
+        private readonly ADAuthTokenProvider _authToken = new();
+        private readonly string _projectDir = Path.GetFullPath(Path.Combine(Environment.CurrentDirectory));
+        //for local
+        //private readonly string _projectDir = Path.GetFullPath(Path.Combine(Environment.CurrentDirectory, "..\\..\\.."));
+
+        [SetUp]
+        public void Setup()
+        {
+            _LUpdatedWebhookEndpoint = new LicenceUpdatedEndpoint();
+        }
+
+        [Test(Description = "WhenValidLUEventInNewEncContentPublishedEventOptions_ThenWebhookReturns200OkResponse"), Order(0)]
```

Comment:
Please rename NewEncContentPublishedEvent to LicenceUpdatedPublishedEventReceived from

WhenValidLUEventInNewEncContentPublishedEventOptions everywhere

Reply:
done